src/App.tsx

```diff
@@ -460,7 +466,7 @@ export const App = ({
 	}
 
 	return (
-		<Column>
+		<div className={commentColumnWrapperStyles}>
```

Comment:
Now that we're not using Column, I wonder if we can merge this div with its child? One less div is always a win